products/paper/paper_info.yaml

```diff
@@ -1,10 +1,8 @@
 ---
 title: "Disparities in Rural Indigenous Community Access to Healthcare in the United States"
-author: Amanda Ricketts
+author: 
 - affiliation: University of Oregon, Sociology
-  name: Aaron Gullickson
-- affiliation: Some Research University
-  name: My co-author
+  name: Amanda Ricketts
```

Comment:
Your name goes under name not author. Author is the over-arching category of which name and affiliation are the things you actually put values in.